packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -143,4 +143,210 @@ class CupertinoIcons {
 
   /// Two right-facing intertwined arrows.
   static const IconData shuffle_thick = IconData(0xf221, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A camera for still photographs. This icon is filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [photo_camera], which is similar, but not filled in.
+  ///  * [video_camera_solid], for the moving picture equivalent.
+  static const IconData photo_camera = IconData(0xf3f5, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A camera for still photographs. This icon is not filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [photo_camera_solid], which is similar, but filled in.
+  ///  * [video_camera], for the moving picture equivalent.
+  static const IconData photo_camera_solid = IconData(0xf3f6, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A camera for moving pictures. This icon is not filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [video_camera_solid], which is similar, but filled in.
+  ///  * [photo_camera], for the still photograph equivalent.
+  static const IconData video_camera = IconData(0xf4cc, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A camera for moving pictures. This icon is filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [video_camera], which is similar, but not filled in.
+  ///  * [photo_camera_solid], for the still photograph equivalent.
+  static const IconData video_camera_solid = IconData(0xf4cd, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A camera containing two circular arrows pointing at each other, which indicate switching. This icon is not filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [switch_camera_solid], which is similar, but filled in.
+  static const IconData switch_camera = IconData(0xf49e, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A camera containing two circular arrows pointing at each other, which indicate switching. This icon is filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [switch_camera], which is similar, but not filled in.
+  static const IconData switch_camera_solid = IconData(0xf49f, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A collection of envelopes (or folders), which store collections of files. This icon is not filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [collections_solid], which is similar, but filled in.
+  static const IconData collections = IconData(0xf3c9, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A collection of envelopes (or folders), which store collections of files. This icon is filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [collections], which is similar, but not filled in.
+  static const IconData collections_solid = IconData(0xf3ca, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A single envelope (or folder), which stores multiple files. This icon is not filled in.
```

Comment:
The definition of envelope says 'a flat paper container with a sealable flap' so this is not it :)

Just say folder.